config/phpro.smartcrud.global.dist.php

```diff
@@ -64,6 +64,9 @@
  *
 array(
     AbstractCrudControllerFactory::FACTORY_NAMESPACE => array(
+        'default' => array(
+            AbstractCrudControllerFactory::CONFIG_VIEW_MODEL_BUILDER => 'App\View\CustomViewModelBuilder',
```

Comment:
@stefliekens 
Can you add the default smartcrud viewbuilder in this configuration file?
This way the implementers will have a working config file if they uncomment these lines.